index.html

```diff
@@ -1,70 +1,8 @@
+---
+layout: default
+title: Screenly Configuration Wizard (Beta)
+---
 
-
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset="UTF-8">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
-  <style>
-  body {
-    background-color: #ffffff;
-  }
-
-  h1 {
-    text-align: center;
-  }
-
-  .row {
-    padding: 0px;
-  }
-
-  .controls {
-    text-indent: 40px;
-    padding: 20px;
-  }
-
-  .form-control {
-    border:1px solid #555555;
-    border-radius: 0px;
-  }
-
-  .form-horizontal {
-    margin-left: 40px;
-  }
-
-  .control-label.col-md-3 {
-    text-align: left;
-  }
-
-  #wifi-fields. #wired-fields, #ntp-fields, #dns-fields {
-    display: none;
-  }
-
-  span.glyphicon:hover {
-    cursor: pointer;
-  }
-
-  #ntp-addbutton , #dns-addbutton, .remove-button  {
-      vertical-align: middle;
-      margin-top: 5px;
-  }
-  </style>
-  <title>Screenly Configuration Wizard (Beta)</title>
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-})(window,document,'script','dataLayer','GTM-M52CH4L');</script>
-  <!-- End Google Tag Manager -->
-</head>
-<body>
-  <!-- Google Tag Manager (noscript) -->
-  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-M52CH4L"
-      height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-  <!-- End Google Tag Manager (noscript) -->
-  <div class="container">
     <div class="row">
       <div class="row">
```

Comment:
Wrong. We use one parent element with classname `.row` following all `.col-*` children. If the idea is to manipulate margins use CSS for so.